index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Ivan Martinez - Portfolio</title>
+   <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+   <header>
+      <nav>
+          <img class="img1" src="resources/image_geometry_1.svg" alt="Mosaic above">
```

Comment:
- [ ] kindly make the logo has a   link and in (`a` tag) has it was required
by the previous reviewer :grinning: 
* This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. The logo (if the design has one) is on the left and has a link (a tag).

> Kindly use the <a> tag for the logo smiley
> 
> This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Reply:
I don't see a logo in the design but I take the up-left image as a logo.

Reply:
@ivanmvh  yees the image is serving as your logo